### DISCUSSION TITLE
->format = Unexpected data found. Unexpected data found. Trailing data

### DISCUSSION THREAD

**Discussion by synsyst** [malzariey/filament-daterangepicker-filter#131] [Q&A]
This method `->format('Y-m-d')` on the filter causes the `error: Unexpected data found. Unexpected data found. Trailing data`

Data in table is a timestampz - the format('Y-m-d') works fine in other components.

full filter:

```
DateRangeFilter::make('created_at')
  ->Label(__('Created at'))
  ->format('Y-m-d')
  ->timePicker24()
  ->separator(' - ')
  ->autoApply(),

```
Anyone know how I'm supposed to format this?

**Comment by malzariey** [comment]
You would need to match the format (Carbon Format) and displayFormat (Java Format)
